### PR TITLE
Add versions of BrainStem and DragonAttenuation with EXT_meshopt_compression

### PR DESCRIPTION
**Pull request by zeux**
BrainStem was compressed with gltfpack using -vpf -cc (floating-point position quantization + EXT_meshopt_compression)

DragonAttenuation was compressed with gltfpack using -cc (fixed-point position quantization + EXT_meshopt_compression)

There's no particular reason to use -vpf for BrainStem but it just serves as a slightly different encoding example that uses a different set of meshopt filters. Note that in BrainStem most of the data is animation data which ends up being compressed fairly well.

Both models render propertly in Three.JS and Babylon.JS but they do require EXT_meshopt_compression so they currently aren't fully validated (although validator emits no errors).

### CODE REVIEW

**Comment by zeux** [comment]
FYI @prideout

**Comment by prideout** [comment]
🥳

**Comment by emackey** [comment]
The new files are in a folder called `glTF-Compressed`.  Given that we already have another compressed folder (`glTF-Draco`), would it make sense to name the new folder `glTF-Meshopt` instead?

**Comment by zeux** [comment]
Makes sense, I've made this change.

File: 2.0/DragonAttenuation/glTF-Meshopt/DragonAttenuation.gltf [hunk 1]
```diff
@@ -0,0 +1 @@
...licRoughness":{"baseColorFactor":[0.921000004,0.639999986,0.064000003,1],"metallicFactor":0,"roughnessFactor":0},"extensions":{"KHR_materials_transmission":{"transmissionFactor":1},"KHR_materials_volume":{"thicknessTexture":{"index":1,"extensions":{"KHR_texture_transform":{"offset":[0,0],"scale":[16.003664,16.003664]}}},"thicknessFactor":2518.09888}}}],"meshes":[{"primitives":[{"attributes":{"POSITION":0,"NORMAL":1,"TEXCOORD_0":2},"indices":3,"material":0}]},{"primitives":[{"attributes":{"POSITION":4,"NORMAL":5,"TEXCOORD_0":6},"indices":7,"material":1,"extensions":{"KHR_materials_variants":{"mappings":[{"material":1,"variants":[0]},{"material":2,"variants":[1]}]}}}]}],"nodes":[{"mesh":0,"translation":[-7.0467,-3.15129995,-9.9399004],"scale":[0.000901473686,0.000901473686,0.000901473686]},{"mesh":1,"translation":[-7.0467,-3.15129995,-9.9399004],"scale":[0.000901473686,0.000901473686,0.000901473686]},{"name":"Dragon","translation":[0,-0.730647981,0],"rotation":[0.707106829,0,0,0.707106829],"scale":[0.25,0.25,0.25],"children":[1]}],"scenes":[{"name":"Scene","nodes":[0,2]}],"scene":0,"extensions":{"KHR_materials_variants":{"variants":[{"name":"Attenuation"},{"name":"Surface Color"}]}}}
```
Comment (bghgary) [inline 2.0/DragonAttenuation/glTF-Meshopt/DragonAttenuation.gltf:1]: I would recommend formatting the JSON instead of a single line for readability.

Comment (zeux) [reply 2.0/DragonAttenuation/glTF-Meshopt/DragonAttenuation.gltf]: Done

**Comment by bghgary** [comment]
For reference, here are links for rendering in Babylon Sandbox:
- https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/adb431c6b6e16244618099c1daef2fb100bb63f6/2.0/BrainStem/glTF-Meshopt/BrainStem.gltf
- https://sandbox.babylonjs.com/?assetUrl=https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/adb431c6b6e16244618099c1daef2fb100bb63f6/2.0/DragonAttenuation/glTF-Meshopt/DragonAttenuation.gltf

**Review by bghgary — APPROVED** [review]

**Review by emackey — APPROVED** [review]

**Comment by emackey** [comment]
Thanks @zeux!

**Pull request merged**